syft/generic/string.py

```diff
@@ -0,0 +1,173 @@
+from typing import List, Tuple
+import syft as sy
+from syft.generic.pointers.string_pointer import StringPointer
+from syft.workers.base import BaseWorker
+from syft.generic.object import AbstractObject
+import abc
+
+
+class String(AbstractObject):
+    def __init__(
+        self,
+        object: object = None,
+        encoding: str = None,
+        errors: str = None,
+        id: int = None,
+        owner: BaseWorker = None,
+        tags: List[str] = None,
+        description: str = None,
+    ):
```

Comment:
Add a dosctring to explain the kwargs (like encoding, errors, etc) which is very useful!

Reply:
Done